lib/features/support/hooks.rb

```diff
@@ -1,13 +1,18 @@
 # frozen_string_literal: true
 
 require 'cucumber'
+require 'fileutils'
 require 'json'
 require 'securerandom'
 require 'selenium-webdriver'
 require 'uri'
 
 AfterConfiguration do |_cucumber_config|
 
+  # Clear out maze_output folder
+  maze_output = File.join(Dir.pwd, 'maze_output')
+  FileUtils.rm_rf(maze_output)
```

Comment:
Doesn't this delete the entire directory? This would break the idea of adding a `.gitignore` to it

This could be something like:

```suggestion
  # Clear out maze_output folder
  maze_output = Dir.glob(File.join(Dir.pwd, 'maze_output', '*'))
  FileUtils.rm_rf(maze_output)
```

Reply:
Good point, nice suggestion.